docs/conf.py

```diff
@@ -234,13 +229,13 @@
     "meth": "tooltip",
     "mod": "tooltip",
     "obj": "tooltip",
-    #
     # roles with std domain
     "confval": "tooltip",
     "hoverxref": "tooltip",
     "ref": "tooltip",
     "term": "tooltip",
 }
+hoverxref_intersphinx = ["numpy", "astropy", "python"]
```

Comment:
We use scipy in places, worth adding?

Reply:
I don't think they mention scipy as supported but I should check as both scipy and numpy are basically the same doc site.